FUNDABLES.md

```diff
@@ -54,6 +54,33 @@ frontend development, testing, DevOps/infrastructure/platform services,
 user experience work, technical writing for end users, project
 management, and community outreach.
 
+#### Port auditwheel to Windows and Mac
+
+The packaging tool [auditwheel](https://github.com/pypa/auditwheel)
+"is a command line tool to facilitate the creation of Python wheel
+packages for Linux (containing pre-compiled binary extensions) that
+are compatible with a wide variety of Linux distributions" and key
+standards. It can inspect a wheel, checking whether it is
+standards-compliant. It can also repair a wheel. If a wheel depends on
+libraries that are not on the system, it can rewrite that wheel and inject
+libraries needed, parsing and rewriting ELF data. It can also repair the
+relevant `manylinux` tag(s) on a wheel.
+
+However, no such utility exists on Windows and Mac OS X, and so
```

Comment:
More nuanced:

- No such utility exists on Windows
- There [is a similar utility for macOS](https://pypi.org/project/delocate/) (which actually predates auditwheel), but it doesn't share code/maintenance/UI

So the advantage of porting auditwheel to Windows is that it would make it *much* easier to make Windows wheels, and the advantage of porting it to macOS is that it would reduce duplication on the packaging maintainers side, and reduce the proliferation of quirky tools that individual package maintainers need to learn about.